msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -403,9 +403,9 @@ The query parameters are:
 
 Example: `https://teams.microsoft.com/l/entity/fe4a8eba-2a31-4737-8e33-e5fae6fee194/tasklist123?webUrl=https://tasklist.example.com/123&TaskList`
 
-### Navigate to an audio or audio-video call
+## Navigate to an audio or audio-video call
 
-You can invoke audio only or audio-video calls to a single user or a group of users, by specifying the call type and the participants. Before placing the call, Teams client prompts a confirmation to make the call. In case of group call, you can call a set of VoIP users and a set of PSTN users in the same deep link invocation.
+You can invoke audio only or audio-video calls to a single user or a group of users, by specifying the call type and the participants. Before placing the call, Teams client prompts a confirmation to make the call. If there is group call, you can call a set of VoIP users and a set of PSTN users in the same deep link invocation.
```

Comment:
Do we need this? The resulting sentence doesn't quite make sense "If there is group call, ..."

Reply:
Edited and updated with "In the case of group call.."

Reply:
@v-ajiths "In the case of a group call"
Please check for missing articles